src/analyze/knip.ts

```diff
@@ -0,0 +1,148 @@
+import {ReportPluginResult} from '../types.js';
+import type {FileSystem} from '../file-system.js';
+import {LocalFileSystem} from '../local-file-system.js';
+import {spawn} from 'node:child_process';
+import {existsSync} from 'node:fs';
+import {join} from 'node:path';
+
+async function which(cmd: string): Promise<string | null> {
+  return new Promise((resolve) => {
+    const proc = spawn('which', [cmd]);
```

Comment:
I don't believe `which` will run on Windows.  Is that a concern?

Reply:
hello @michaelfaith, nice catch, but i don't think it is a concern right now, since we are in early development 😁

Reply:
also we stumble upon a big rabbit hole with this one that's why it has become stale.